src/Foundatio.RabbitMQ/Messaging/RabbitMQMessageBus.cs

```diff
@@ -293,24 +285,21 @@ private bool CreateDelayedExchange(IModel model)
             // Disclaimer : https://github.com/rabbitmq/rabbitmq-delayed-message-exchange/
             // Please read the *Performance Impact* of the delayed exchange type.
             var args = new Dictionary<string, object> { { "x-delayed-type", ExchangeType.Fanout } };
-            model.ExchangeDeclare(_options.Topic, "x-delayed-message", _options.IsDurable, false, args);
+            await channel.ExchangeDeclareAsync(_options.Topic, "x-delayed-message", _options.IsDurable, false, args).AnyContext();
         }
         catch (OperationInterruptedException ex)
         {
-            if (ex.ShutdownReason.ReplyCode == 503)
-            {
-                if (_logger.IsEnabled(LogLevel.Information)) _logger.LogInformation(ex, "Not able to create x-delayed-type exchange");
-                success = false;
-            }
+            _logger.LogInformation(ex, "Unable to create x-delayed-type exchange: {Message}", ex.Message);
```

Comment:
There is a bug here where success could be true even though it threw an exception which I think is wrong. I can change this back. But if it can't be thrown in the lasted rabbit 4.x the status reply code is 406. I'm not sure why it changed but this seems like a safer implementation. 

cc @ejsmith @twehner